boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -0,0 +1,107 @@
+.. zephyr:board:: pocketbeagle_2
+
+Overview
+********
+
+PocketBeagle 2 is a computational platform powered by TI AM62x SoC (there are two
+revisions, AM6232 and AM6254).
+
+The board configuration provides support for the ARM Cortex-M4F MCU core and the
+following features:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
```

Comment:
UART? MBOX? GPIO?

Reply:
@Ayush1325 we will correct those, likely those were not updated when the peripheral support were updated after the platform support was added.

Reply:
As suggested by @kartben I have dropped the features from here.